tensorbay/sensor/intrinsics.py

```diff
@@ -56,9 +56,9 @@ class CameraMatrix(ReprMixin):
 
```

Comment:
your commit message and name of pull request is wrong, please check with @Lee-000